build/ferretdb/evaluation/supervisord.conf

```diff
@@ -0,0 +1,18 @@
+[supervisord]
+nodaemon=true
+logfile=/dev/stderr
+logfile_maxbytes=0
+
+[program:postgres]
+command=/usr/local/bin/docker-entrypoint.sh postgres
+; fast shutdown https://www.postgresql.org/docs/current/server-shutdown.html
+stopsignal=INT
+stdout_logfile=/dev/stderr
+stdout_logfile_maxbytes=0
+redirect_stderr=true
+
+[program:ferretdb]
```

Comment:
We probably could add a dependency there

Reply:
Added priority which sets the starting order of processes.